CHANGELOG.md

```diff
@@ -2,22 +2,20 @@
 
 ## 1.23.0 (TBD)
 
-### Snowpark pandas API Updates
+### Snowpark Python API Updates
 
 #### New Features
 
 - Added the following new functions in `snowflake.snowpark.functions`:
   - `make_interval`
-- Added support for using Snowflake Interval constants with `Window.range_between()` when the order by column is TIMESTAMP or DATE type
+- Added support for using Snowflake Interval constants with `Window.range_between()` when the order by column is TIMESTAMP or DATE type.
+- Added support for file writes. This feature is currently in private preview.
```

Comment:
So these changes to CHANGELOG are fixing multiple mixups from previous PRs? (Kind of surprised they were that many!)

Reply:
Yeah we didn't have the Snowpark Python header after cutting the release so got mixed in